CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on Keep a Changelog (https://keepachangelog.com/en/1.0.0/),
 
+## [4.1.1] - [unreleased]
```

Comment:
It's great that you thought about the changelog. However we are already on version 5.0.0 unreleased.